client/app/components/visualizations/editor/ContextHelp.jsx

```diff
@@ -41,7 +41,7 @@ function DateTimeFormatSpecs() {
       title={
         <React.Fragment>
           Formatting Dates and Times
-          <i className="fa fa-external-link m-l-5" />
+          <i className="fa fa-external-link" style={{ marginLeft: 5 }} />
```

Comment:
@kravets-levko I do try to avoid using inline styling, but replacing those isolated cases with only one css rule with it was really tempting

Reply:
Brilliant! That's what I always wanted to do but never had a chance :smile: